tests/qconvolutional_test.py

```diff
@@ -60,7 +57,7 @@ def test_qnetwork():
       strides=(2, 2),
       depthwise_quantizer=binary(alpha=1.0),
       pointwise_quantizer=quantized_bits(4, 0, 1, alpha=1.0),
-      depthwise_activation=quantized_bits(6, 2, 1, alpha=1.0),
```

Comment:
The depthwise activation here is necessary to maintain quantized activations. We should also add `pointwise activation` here by using the `activation` param (which should probably be renamed to `pointwise_activation`)

Reply:
This is related to the discussion above.